### DISCUSSION TITLE
Cannot login to account: error 404 page not found

### DISCUSSION THREAD

**Discussion by Kikkererwt** [opengisch/QField#3331] [Bugs]
I think I know why, but let me try to explain how I kinda messed up this account. 
Whenever I try to loging, I get an errorpage 404 page not found.

How it happened: 
I first registered my original account, with no problems. I created some projects, tested them, approved them.
Then I created an organization with my original account. No problems there.
Next I wanted to add a member to the organization, so I chose my official e-mailaddress to add as member.
I got an e-mail asking if I wanted to join, so I clicked it. But then I don't quite remember what happened. I think I filled in my data to create a new account, and I was stopped saying the account already existed. So I let it sent a mail to change password, and I did, but then I got the error page 404. And now whenever I login, I get the error page, and I can't log out because it alwayse reverts back to the 404 page.

Is there any way to delete this account completely and try again?

**Comment by suricactus** [comment]
If you give me your username I can try sorting things out for you. :)

**Reply by Kikkererwt** [reply]
I believe the account name that is getting the errorpage is called "Bodem". I have the e-mailaddress but I don't know how to send a pm, if that's even possible on here. Thanks in advance!!

Edit: no, maybe the account name is "PCM".

**Reply by Kikkererwt** [reply]
I don't know if you did anything by now. But I tried logging in and couln't anymore. When I reset my password, and login with the new password afterwards, it gives me an error, saying my e-mail and password do not match. 
Is there any way to fix this? Maybe delete the account and let me make a new one with the same e-mail?

Edit: fixed it. PCM was account linked to an organization, and not to an actual account created. I removed the organization and then I could create a new account with my e-mail address. So problem solved!
